Valet/VALSecureEnclaveValet.m

```diff
@@ -223,9 +223,20 @@ - (nullable instancetype)initWithSharedAccessGroupIdentifier:(nonnull NSString *
     self = [super initWithSharedAccessGroupIdentifier:sharedAccessGroupIdentifier accessibility:accessibility];
     if (self != nil) {
         SEL const backwardsCompatibleInitializer = @selector(initWithSharedAccessGroupIdentifier:accessibility:);
+#if TARGET_IPHONE_SIMULATOR
```

Comment:
`TARGET_IPHONE_SIMULATOR` is deprecated, replaced by `TARGET_OS_SIMULATOR`. But I'm pretty sure that `TARGET_OS_SIMULATOR` was introduced in Xcode 7, and we're still supporting Xcode 6. Since I didn't want to bump our minimum Xcode version over this, I'm using the old value (which still works)